README.md

```diff
@@ -86,12 +93,10 @@ class StableDiffusion(MsgpackMixin, Worker):
             dummy_file = BytesIO()
             img.save(dummy_file, format="JPEG")
             images.append(dummy_file.getbuffer())
-        # need to return the same number of images in the same request order
-        # `len(data) == len(images)`
         return images
 ```
 
-Finally, we append the worker to the server to construct a *single-stage* workflow, and specify the number of processes we want it to run in parallel. Then run the server.
+Finally, we append the worker to the server to construct a *single-stage* workflow (multiple stages can be [pipelined](https://en.wikipedia.org/wiki/Pipeline_(computing)) to further boost the throughput, see [this example](https://mosecorg.github.io/mosec/example/pytorch/#computer-vision)), and specify the number of processes we want it to run in parallel (`num=1`), and the maximum batch size (`max_batch_size=16`, the maximum number of requests dynamic batching will accumulate before timeout).
```

Comment:
The last `timeout` may confuse users. I think it's better to clarify the relation with `--wait` here.